flocker/acceptance/testtools.py

```diff
@@ -897,11 +901,39 @@ def release_all(leases):
                     Exception("Timed out cleaning up leases"),
                 )
 
+        def cleanup_volumes():
```

Comment:
It shouldn't; the previous step should have already cleaned everything up, except for that exercise the new functionality that creates volumes that aren't reported as datasets.